### PR TITLE
docs: add entity names as comments to skeleton files

### PR DESCRIPTION
**Pull request by Uberbagel**
Per Darxo's request, which I agree with.

### CODE REVIEW

File: mod_reforged/hooks/entity/tactical/enemies/skeleton_light.nut [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//Ancient Auxiliary
```
Comment (Darxo) [inline mod_reforged/hooks/entity/tactical/enemies/skeleton_light.nut:1]: Using the very first line for that feels a bit odd.
Maybe it should go on the right side of the HooksMod/inherit line? Separated by a single tab or space
I would also add a single space bar after the slashes.

e.g.:
`::Reforged.HooksMod.hook("scripts/entity/tactical/enemies/skeleton_light", function(q) { // Ancient Auxiliary`

File: scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//Ancient Praetorian
```
Comment (Darxo) [inline scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut:1]: Using the very first line for that feels a bit odd.
Maybe it should go on the right side of the inherit line? Separated by a single tab or space
I would also add a single space bar after the slashes.

e.g.:
`this.rf_skeleton_heavy_lesser <- ::inherit("scripts/entity/tactical/skeleton", { // Ancient Praetorian`

Comment (Uberbagel) [reply scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut:1]: I'm neutral as to which is nicer. Will see what Midas thinks

Comment (LordMidas) [reply scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut:1]: I prefer the comment on the first line, as it is much more visible the moment you open the file. But I agree with Darxo that there should be a space after the slashes so `// Ancient Praetorian`

Comment (Darxo) [reply scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut:1]: Alright, let's leave it in the first row with an additional space each

Comment (Uberbagel) [reply scripts/entity/tactical/enemies/rf_skeleton_heavy_lesser.nut]: Done

**Review by Darxo — APPROVED** [review]

**Review by LordMidas — APPROVED** [review]

**Pull request merged**